rm-datasource/src/main/java/io/seata/rm/datasource/undo/AbstractUndoLogManager.java

```diff
@@ -288,7 +281,7 @@ public void undo(DataSourceProxy dataSourceProxy, String xid, long branchId) thr
                             Collections.reverse(sqlUndoLogs);
                         }
                         for (SQLUndoLog sqlUndoLog : sqlUndoLogs) {
-                            TableMeta tableMeta = TableMetaCacheFactory.getTableMetaCache(dataSourceProxy).getTableMeta(
+                            TableMeta tableMeta = TableMetaCacheFactory.getTableMetaCache(dataSourceProxy.getDbType()).getTableMeta(
```

Comment:
dataSourceProxy.getDbType() return SqlParser.getDbType, but in other places is returned we define constants, whether there is the risk of inconsistent?

Reply:
Yes, there is a problem. It has been corrected